google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -385,7 +385,14 @@ error.dialog.disable.plugin.norestart=After disabling the plugin please restart
 error.dialog.disable.plugin.title=Disable Plugin
 error.dialog.disable.plugin.action.disable=&Disable
 error.dialog.disable.plugin.action.disableAndRestart=Disable and &Restart
+
 account.plugin.removal.requires.restart.title=Google Cloud Tools: Please restart your IDE again.
 account.plugin.removal.requires.restart.text=The Google Account plugin is no longer required and \
   has been uninstalled.  Please restart your IDE for it to take effect.  If you cancel you will \
   experience problems using the Google Cloud Tools plugin. \n\nOK to restart?
+
+project.selector.no.selected.project=No Google Cloud project selected.
```

Comment:
For this state, what do you think about phrasing it as an action for the user, e.g.:

"Select a Google Cloud project".

Maybe that is clearer?

Reply:
I think this might be better, thanks